osu.Game/Screens/Multi/RealtimeMultiplayer/RealtimeMatchSongSelect.cs

```diff
@@ -58,7 +58,7 @@ protected override bool OnStart()
 
                 client.ChangeSettings(item: item).ContinueWith(t =>
                 {
-                    return Schedule(() =>
+                    Schedule(() =>
```

Comment:
Is there any semantic difference with this change, other than that the underlying task changes type (from `Task<ScheduledDelegate>` to `Task`)? Not really bothered either way, just curious.

Reply:
just couldn't see a reason fort he returning of the schedule.